pkg/admissioncontroller/apis/config/validation/admissioncontrollerconfiguration.go

```diff
@@ -27,6 +28,14 @@ import (
 func ValidateAdmissionControllerConfiguration(config *apisconfig.AdmissionControllerConfiguration) field.ErrorList {
 	allErrs := field.ErrorList{}
 
+	if config.LogLevel != logger.DebugLevel && config.LogLevel != logger.InfoLevel && config.LogLevel != logger.ErrorLevel {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("logLevel"), config.LogLevel, "name must be one of 'debug', 'info' or 'error'"))
+	}
+
+	if config.LogFormat != logger.FormatJSON && config.LogFormat != logger.FormatText {
+		allErrs = append(allErrs, field.Invalid(field.NewPath("logFormat"), config.LogLevel, "name must be either 'json' or 'text'"))
+	}
```

Comment:
You could reuse the already defined lists with allowed log levels and formats:

```suggestion
	if !sets.NewString(logger.AllLogLevels...).Has(config.LogLevel) {
		allErrs = append(allErrs, field.NotSupported(field.NewPath("logLevel"), config.LogLevel, logger.AllLogLevels))
	}

	if !sets.NewString(logger.AllLogFormats...).Has(config.LogFormat) {
		allErrs = append(allErrs, field.NotSupported(field.NewPath("logFormat"), config.LogFormat, logger.AllLogFormats))
	}
```